docs/source/user-guide/general/drift_guide.rst

```diff
@@ -0,0 +1,301 @@
+* For discrete or categorical distributions - `Population Stability Index (PSI) <https://www.lexjansen.com/wuss/2017/47_Final_Paper_PDF.pdf>`__ or `Cramer's V <https://en.wikipedia.org/wiki/Cram%C3%A9r%27s_V>`__
+
+These methods have the advantage of being simple to use and produce explainable results. However, they are limited by
+checking each feature one at a time, and cannot detect drift in the relations between features. Also, these methods
+will usually detect drift multiple times if it occurs in several features.
+
+.. _drift_detection_by_domain_classifier:
+Detection by Domain Classifier
+------------------------------
+
+Training a `Domain Classifier <https://arxiv.org/abs/2004.03045>`__ is a method to detect multivariate drift, meaning that it
+can run on several variables, and even on the whole dataset.
+This is done by training a model to classify if a sample came from the train dataset or the new (test or production) dataset.
+If the classifier can easily predict which sample is from which dataset, it would mean that there are significant differences between these datasets.
+
+The main advantage of this method is that it can also uncover covariate drift, meaning drift in the data that does not
+affect the distribution of each individual variable, but does affect the relationship between them.
+
+In deepchecks (in checks :doc:`Whole Dataset Drift</checks_gallery/tabular/train_test_validation/plot_whole_dataset_drift>` and
+:doc:`Image Dataset Drift</checks_gallery/vision/train_test_validation/plot_image_dataset_drift>`) we merge
+the train and the test sets, and assign label 0 to samples that come from the training set, and 1 to those who are
+from the test set. Then, we train a binary classifer of type
+`Histogram-based Gradient Boosting Classification Tree
+<https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.HistGradientBoostingClassifier.html>`__.
+We normalize the AUC score of this classifier and use it as the drift score, as the higher the AUC, the better the model,
+meaning the datasets are significantly different.
+
+
+How Can I Use Deepchecks to Detect Drift?
+=========================================
+
+Deepchecks can test your data for both concept drift and data drift, by using a variety of methods.
+
+Tabular Data
+------------
+
+To detect `data <#data-drift>`__ or `concept drift <#concept-drift>`__, deepchecks offers the
+:doc:`Feature Drift check </checks_gallery/tabular/train_test_validation/plot_train_test_feature_drift>` which uses
+`univariate measures <#detection-by-univariate-measure>`__ and the :doc:`Whole Dataset Drift check</checks_gallery/tabular/train_test_validation/plot_whole_dataset_drift>`
+which uses a `domain classifier <#detection-by-domain-classifier>`__ in order to detect multivariate drift.
+
+For drift in your label's distribution, deepchecks offers the :doc:`Label Drift check </checks_gallery/tabular/train_test_validation/plot_train_test_label_drift>`,
```

Comment:
If we way nothing of label drift previously, I'm for saying something about it's usefulness here.